lib/ReactViews/Analytics/InvokeFunction.jsx

```diff
@@ -32,6 +32,10 @@ const InvokeFunction = React.createClass({
         this.initializeParameters(nextProps);
     },
 
+    componentWillUpdate(nextProps, nextState) {
+        this.initializeParameters(nextProps);
+    },
+
```

Comment:
Also, I haven't seen `knockout` tracking on a view variable before... I guess there's no reason we can't do it. But since that view variable is just a reworking of a variable defined on a model, wouldn't it be cleaner to track the model variable instead? Maybe this is a question for @kring ... :-)